core/framework/comparator_test.go

```diff
@@ -16,9 +16,9 @@ package framework
 
 import (
 	"reflect"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
please harmonize the naming of this package throughout go files

Reply:
done. All go client in kogito-operator now refers `"sigs.k8s.io/controller-runtime/pkg/client"` as `client`.